packages/web/src/scss/components/UNSTABLE_Section/_UNSTABLE_Section.scss

```diff
@@ -0,0 +1,9 @@
+.UNSTABLE_Section:not(:last-child) {
+    margin-bottom: 2.5rem;
```

Comment:
Is it deliberately stated in rem?  Couldn't we use tokens?

Reply:
This is just a refactor from `docs-Section` to `UNSTABLE_Section`. So I have no intention to change this now. But there will be another iteration after the discovery of the Section component will be done.